src/Compilers/CSharp/Portable/Binder/Binder.QueryUnboundLambdaState.cs

```diff
@@ -32,6 +32,7 @@ public QueryUnboundLambdaState(Binder binder, RangeVariableMap rangeVariableMap,
             public override int ParameterCount { get { return _parameters.Length; } }
             public override bool IsAsync { get { return false; } }
             public override RefKind RefKind(int index) { return Microsoft.CodeAnalysis.RefKind.None; }
+            public override bool IsReadOnly(int index) => false;
```

Comment:
I assume this is about parameters, I think the name should make it clear. For example, IsReadonlyParameter.